js/index.html

```diff
@@ -4,7 +4,7 @@
     <title>Sample Page | UX capture</title>
 
     <!-- don't do this in production, but inline ux-capture.min.js instead -->
-    <script src="ux-capture.js"></script>
+    <script src="ux-capture.min.js"></script>
```

Comment:
seems like you're using min, so remove the comment above?

Reply:
Not exactly, inlining is still needed, just no need to point out the file name.